### PR TITLE
feat: support namespace resources

### PR DESCRIPTION
**Pull request by mattiascockburn**
* add create_namespace/delete_namespace action
* add namespace_exists probe

### CODE REVIEW

File: chaosk8s/namespace/actions.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        ns = {"apiVersion": "v1", "kind": "Namespace", "metadata": {"name": name}}
+    else:
+        raise ActivityFailed("You need to either specify name or spec_path")
+    v1 = client.CoreV1Api(api)
+    v1.create_namespace(body=ns)
+
+
+def delete_namespace(name: str, secrets: Secrets = None):
+    """
+    Remove the given secret
```
Comment (Lawouach) [inline chaosk8s/namespace/actions.py:41]: Wrong copy/paste ;)

**Review by Lawouach — CHANGES_REQUESTED** [review]
Nice.

A small change: can you add these actions/probes to the discover function in the top-level __init__.py module please? They will be discovered by the documentation builder that way.

File: chaosk8s/namespace/actions.py (hunk 1 above)
Comment (mattiascockburn) [reply chaosk8s/namespace/actions.py]: 🙃

**Review by Lawouach — APPROVED** [review]
LGTM

**Comment by Lawouach** [comment]
You'll have to rebase  this branch and resolve the conflict and it'll be good to go @mattiascockburn

**Pull request merged**